zuul-core/src/main/java/com/netflix/zuul/niws/RequestAttempt.java

```diff
@@ -325,6 +326,9 @@ else if (t instanceof OutboundException) {
                 error = t.getMessage();
                 exceptionType = t.getClass().getSimpleName();
             }
+            if (t.getCause() != null) {
+                cause = t.getCause().toString();
```

Comment:
This should be Throwables.getStackTraceAsString, which correctly expands the causes and suppressed exceptions and stacks.

Reply:
Good point. I think what we really want here is `getMessage()`.